src/planning/planning-modality-handler.ts

```diff
@@ -25,7 +25,7 @@ export class PlanningModalityHandler extends ModalityHandler{
   }
 
   static createHeadlessHandler(): PlanningModalityHandler {
-    return new PlanningModalityHandler(MockSlotDomConsumer, MockSuggestDomConsumer);
+    return new PlanningModalityHandler(HeadlessSlotDomConsumer, MockSuggestDomConsumer);
```

Comment:
should we do the same for MockSuggestDomConsumer? fine to move it in a follow up PR if you prefer, but the use seems similar.

Reply:
It does appear to be almost identical. But I think I'd rather do it in my very next PR, probably from Sydney next week, so that I don't leave anything partly done between now and then.